site/docs/v1/tech/account-management/troubleshooting.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: doc
+title: Account Management - Troubleshooting
+description: Managing Accounts - Troubleshooting Steps
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Troubleshooting
+
+Not able to get your account management page up and running?  Below are two common popup warnings and how to fix them.
+
+* <<Not Registered>>
+* <<Registered But Configuration Needed>>
+
+=== Not Registered
+
+Please note, if the user is not registered to the application, they will not be able to self serve from the account management page.
```

Comment:
"they will not be able to self serve from the account management page." -> maybe 
"they will not be able to manage their account information from these pages."